tokio/src/local.rs

```diff
@@ -0,0 +1,374 @@
+//! Utilities for running `!Send` futures on the current thread.
```

Comment:
Is it intentional that this file and `tokio/src/executor/local.rs` are nearly identical?

Reply:
Nope, that's my fault; the duplicate file results from moving it to a new location, realizing it wouldn't work since it used module-private APIs, and then forgetting to commit the change back.